tuhi/png.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+
+from gi.repository import GObject
+import cairo
+
+class JsonPng(GObject.Object):
+
+    def __init__(self, json, orientation, filename, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.json = json
+        self.timestamp = json['timestamp']
+        self.filename = filename
+        self.orientation = orientation.lower()
+        self._convert()
+
+    def _convert(self):
+        js = self.json
+        dimensions = js['dimensions']
+        if dimensions == [0, 0]:
+            width, height = 100, 100
+        else:
+            # Original dimensions are too big for SVG Standard
+            # so we normalize them
+            width, height = dimensions[0] // 100, dimensions[1] // 100
+
+        if self.orientation in ['portrait', 'reverse-portrait']:
+            size = (height, width)
+        else:
+            size = (width, height )
```

Comment:
space before ` )`. btw, run `flake-8` and that should complain about this. Or install `flake-8` and run `ninja test` and that does the same.